nl-core/StructureDefinitions/nl-core-SOAPReport.xml

```diff
@@ -0,0 +1,92 @@
+  <purpose value="A derived profile from [zib-SOAPReport](http://nictiz.nl/fhir/StructureDefinition/zib-SOAPReport) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-soapreport-v1.3-2024EN" />
+    <uri value="https://zibs.nl/wiki/SOAPReport-v1.3(2024EN)" />
+    <name value="zib SOAPReport-v1.3(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Composition" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-SOAPReport" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Composition">
+      <path value="Composition" />
+      <comment value="Zib SOAPReport is implemented using two profiles: this profile implements the SOAPReport concept of the zib, while &lt;http://nictiz.nl/fhir/StructureDefinition/nl-core-SOAPReport.SOAPLine&gt; represents the SOAPLine concept of the zib which is referenced by `.section.entry`. &#xD;&#xA;The Composition resource SHOULD hold a summary in the `.section.text` elements of the information found in the referenced SOAPReport.SOAPLine profile." />
+      <alias value="nl-core-SOAPReport" />
+    </element>
+    <element id="Composition.subject">
+      <path value="Composition.subject" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Resource" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="Composition.encounter">
+      <path value="Composition.encounter" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Encounter" />
```

Comment:
Why do we add this reference here (was already the case in 2020) without a mapping to an information standard or use case?